[2024-03-09T15:43:48.629+0530] {processor.py:161} INFO - Started process (PID=4633) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:43:48.630+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:43:48.631+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:43:48.631+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:43:48.687+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:43:48.686+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:43:48.687+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:43:48.723+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.097 seconds
[2024-03-09T15:44:52.736+0530] {processor.py:161} INFO - Started process (PID=4691) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:44:52.738+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:44:52.739+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:44:52.739+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:44:52.799+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:44:52.798+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:44:52.800+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:44:52.863+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.130 seconds
[2024-03-09T15:45:47.743+0530] {processor.py:161} INFO - Started process (PID=4751) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:45:47.746+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:45:47.747+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:45:47.747+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:45:47.806+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:45:47.805+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:45:47.806+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:45:47.839+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.099 seconds
[2024-03-09T15:46:35.963+0530] {processor.py:161} INFO - Started process (PID=4795) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:46:35.964+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:46:35.965+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:46:35.965+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:46:36.049+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:46:36.037+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:46:36.051+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:46:36.098+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.139 seconds
[2024-03-09T15:47:18.281+0530] {processor.py:161} INFO - Started process (PID=4836) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:47:18.284+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:47:18.285+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:47:18.285+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:47:18.332+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:47:18.330+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:47:18.332+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:47:18.365+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.088 seconds
[2024-03-09T15:48:23.774+0530] {processor.py:161} INFO - Started process (PID=4914) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:48:23.776+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:48:23.777+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:48:23.777+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:48:23.831+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:48:23.829+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:48:23.831+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:48:23.865+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.094 seconds
[2024-03-09T15:49:13.231+0530] {processor.py:161} INFO - Started process (PID=4978) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:49:13.234+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:49:13.235+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:13.235+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:49:13.313+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:49:13.311+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:49:13.313+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:49:13.347+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.119 seconds
[2024-03-09T15:50:04.729+0530] {processor.py:161} INFO - Started process (PID=5036) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:50:04.731+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:50:04.732+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:50:04.732+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:50:04.791+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:50:04.789+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:50:04.791+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:50:04.826+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.100 seconds
[2024-03-09T15:50:49.625+0530] {processor.py:161} INFO - Started process (PID=5082) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:50:49.629+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:50:49.630+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:50:49.629+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:50:49.679+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:50:49.677+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:50:49.679+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:50:49.711+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.090 seconds
[2024-03-09T15:51:37.311+0530] {processor.py:161} INFO - Started process (PID=5134) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:51:37.314+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:51:37.315+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:51:37.315+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:51:37.363+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:51:37.361+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:51:37.363+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:51:37.397+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.089 seconds
[2024-03-09T15:52:13.722+0530] {processor.py:161} INFO - Started process (PID=5173) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:52:13.725+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:52:13.727+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:52:13.726+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:52:13.785+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:52:13.783+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:52:13.785+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:52:13.818+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.101 seconds
[2024-03-09T15:53:05.558+0530] {processor.py:161} INFO - Started process (PID=5225) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:53:05.561+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:53:05.562+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:05.562+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:53:05.624+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:05.622+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:53:05.625+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:53:05.656+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.102 seconds
[2024-03-09T15:53:58.382+0530] {processor.py:161} INFO - Started process (PID=5280) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:53:58.384+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:53:58.385+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:58.385+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:53:58.445+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:53:58.443+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:53:58.445+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:53:58.483+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.103 seconds
[2024-03-09T15:54:34.984+0530] {processor.py:161} INFO - Started process (PID=5317) to work on /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:54:34.987+0530] {processor.py:830} INFO - Processing file /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-03-09T15:54:34.989+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:54:34.988+0530] {dagbag.py:540} INFO - Filling up the DagBag from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:54:35.050+0530] {logging_mixin.py:188} INFO - [2024-03-09T15:54:35.048+0530] {dagbag.py:350} ERROR - Failed to import: /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/kai/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 591, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-03-09T15:54:35.050+0530] {processor.py:842} WARNING - No viable dags retrieved from /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py
[2024-03-09T15:54:35.091+0530] {processor.py:183} INFO - Processing /home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_branch_operator.py took 0.111 seconds
